packages/material-ui/src/Select/Select.test.js

```diff
@@ -105,5 +105,17 @@ describe('<Select />', () => {
       );
       assert.strictEqual(wrapper.find(`.${classes.select}`).text(), 'Twenty');
     });
+
+    it('should render a placeholder when the value is missing', () => {
```

Comment:
I think a visual regression test is better suited for this one. The current test doesn't really do anything other than explain the implementation.

Reply:
Sounds like a great idea. I'm changing it.